### ISSUE TITLE
remove "node_modules" folder

### ISSUE THREAD

**Issue by otanim** [Mila152/node-rest-users#1]
You shouldn't keep external/dynamically generated packages/files/folders inside of your repository, for more info read this [tutorial](https://www.atlassian.com/git/tutorials/saving-changes/gitignore).

Make sure to exclude from your repository the following file/folders:

- node_modules/
- nodemon.json

Then use #2 issue's 5th clause to wipe the entire commit history.

**Comment by otanim** [comment]
I can see that you are a Miss of a culture :)
the same #2's comment applies to this issue as well ;)

**Comment by otanim** [comment]
I can confirm that this is correctly done, however, you created another issue, you included config/ folder inside of .gitignore folder, which you shouldn't have done.

**Comment by Mila152** [comment]
but the credentials are under config/

**Comment by otanim** [comment]
yes, that's right, check Github's source codes, that folder is... missing.
